### PR TITLE
Decision about Unit Prefixes

### PR DESCRIPTION
**Pull request by tobru**
## Checklist

- [X] Try to isolate changes into separate PRs (to build a better changelog).
- [X] Categorize the PR by setting a good title and adding one of the labels:
      `change`, `decision`, `requirement/quality`, `requirement/functional`, `dependency`
      as they show up in the changelog

### CODE REVIEW

**Comment by tobru** [comment]
Relates to https://github.com/appuio/appuio-cloud-docs/pull/49

File: docs/modules/ROOT/pages/explanation/decisions/unit-prefixes.adoc [hunk 1]
```diff
@@ -0,0 +1,22 @@
+= Unit prefixes
+
+== Problem
+
+On a low-level we work with bytes.
+Representing bytes to a human isn't practical, therefor a unit prefix has to be used to make it better understandable.
+
```
Comment (ccremer) [inline docs/modules/ROOT/pages/explanation/decisions/unit-prefixes.adoc:7]: Maybe extend the problem desc that this decision would also apply to product specifications and marketing and maybe other places?

**Review by ccremer — COMMENTED** [review]
I do not see any actual impact on the architecture. I merely see this decision as a form of definition that affects product documentation and maybe API specifications and maybe marketing.

Why not simply update https://products.docs.vshn.ch/products/appuio/cloud/price_model.html?

Edit: https://github.com/appuio/appuio-cloud-docs/pull/49 should be enough IMO

File: docs/modules/ROOT/pages/explanation/decisions/unit-prefixes.adoc (hunk 1 above)
Comment (tobru) [reply docs/modules/ROOT/pages/explanation/decisions/unit-prefixes.adoc]: Done.

**Comment by tobru** [comment]
The impact is given, as we have several places where we need to present "bytes" to a human: The Portal, Invoices, ... And as it already caused confusion because this wasn't documented, I think it is important that we write this down.

**Review by corvus-ch — APPROVED** [review]

**Comment by ccremer** [comment]
> The impact is given, as we have several places where we need to present "bytes" to a human: The Portal, Invoices, ... And as it already caused confusion because this wasn't documented, I think it is important that we write this down.

Yes, there's an impact in various places like UI and invoices and we should write it down. It just doesn't have an impact _on the architecture_. I'ts more like a business decision. Displaying GiB instead of GB doesn't fundamentally change how core APPUiO Cloud works.

I propose to add a few sentences [here](https://kb.vshn.ch/appuio-cloud/references/architecture/metrics-of-interest.html#_metric_memory_usage) or  [here](https://kb.vshn.ch/appuio-cloud/references/architecture/metering.html)

**Comment by tobru** [comment]
As I don't see another good place to document it, I go forward and merge it. No harm will be done by that if we have to move it to a different place.

**Pull request merged**